stdlib/public/core/UnsafeBufferPointer.swift.gyb

```diff
@@ -197,19 +252,25 @@ public struct Unsafe${Mutable}BufferPointer<Element>
     _end = start.map { $0 + count }
   }
 
-  /// Returns an iterator over the elements of this sequence.
+  /// Returns an iterator over the elements of this buffer.
   ///
-  /// - Complexity: O(1).
+  /// - Returns: An iterator over the elements of this buffer.
```

Comment:
That's the expected complexity for `makeIterator()`—we only document when it deviates.